js/renovation/ui/grids/data_grid/common/data_grid_props.tsx

```diff
@@ -1018,7 +1018,7 @@ export class DataGridStateStoring {
 @ComponentBindings()
 export class DataGridFilterPanel {
   @Event()
-  customizeText?: (e: FilterPanelCustomizeTextArg<DxDataGrid>) => string;
+  customizeText?: (e: FilterPanelCustomizeTextArg) => string;
```

Comment:
why removing generic?

But anyway no need to change cause we should have deleted `renovation/ui/grids/(data_grid, grid_base)/*` files long time ago

Reply:
This `FilterPanelCustomizeTextArg` is DataGrid-specific:
```ts
/** @public */
export type FilterPanelCustomizeTextArg = ComponentFilterPanelCustomizeTextArg<dxDataGrid>;
```